Detectors/TPC/reconstruction/run/testRawReader.cxx

```diff
@@ -0,0 +1,95 @@
+///
+/// @file   testRawReader.cxx
```

Comment:
I think this is a development tool. @KlewinS should comment. For a unit test also a raw data file would be needed. I don't that this is what we want for a unit test.